config-model/src/main/java/com/yahoo/vespa/model/application/validation/PagedAttributesRemoteStorageValidator.java

```diff
@@ -56,11 +57,12 @@ private static void validatePagedAttributes(DeployLogger logger, String clusterN
                 .formatted(clusterName, join(fields)));
     }
 
-    private static String join(Set<Attribute> fields) {
-        return fields.stream()
+    private static String join(List<Attribute> fields) {
+        var ret = fields.stream()
```

Comment:
add `.limit(10)` to actually limit also?

Reply:
Thanks, f**ked up commit, so had to redo it, apparently lost the most important line when doing that.